components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__gene-variants.xml

```diff
@@ -137,16 +138,19 @@
         var infoTool = new Element('span', {'class' : 'fa fa-info-circle xHelpButton gene-info', 'title' : geneNames[i]});
         var icon_helpController = new PhenoTips.widgets.HelpButton(infoTool);
         var triangle = new Element('span', {'class': "fa fa-exclamation-triangle", title: '$services.localization.render('phenotips.patientSheetCode.termSuggest.nonStandardGenesymbol')'}).update(' ');
-        if (isSuggestSelected)
+        if (isSuggestSelected) {
+          element.removeClassName(nonstandard);
           element.insert({after: infoTool});
-        else
+        } else {
+          element.addClassName(nonstandard);
           element.up().insert({top: triangle});
```

Comment:
@buske OK, but are we all fine with the popover that shows up when clicking the info button for such an entry? (See screenshot below)
 
![ensgxxx](https://cloud.githubusercontent.com/assets/651980/19942841/7c9a46e0-a10b-11e6-98a1-e16580ed6d13.png)